Magento_Wishlist/web/js/view/custom-wishlist.js

```diff
@@ -0,0 +1,33 @@
+define([
+  'uiComponent',
+  'ko',
+  'jquery',
+  'Magento_Customer/js/customer-data',
+  'Magento_Customer/js/model/customer',
+], (Component, ko, $, customerData, customer) => {
+  return Component.extend({
+    defaults: {
+      template: 'Magento_Wishlist/custom-wishlist',
+      totalItems: 0,
+      wishlistUrl: '',
+      isLoggedIn: customer.isLoggedIn(),
```

Comment:
two notes
1. you don't have to call the observable `isLoggedIn` from `customer` because you just get this value right now, but if it'll be updated, you won't see the changes, so use only the linking here
```js
isLoggedIn: customer.isLoggedIn // don't call it as a function
```

2. `Magento_Customer/js/model/customer` is only for checkout. for any other pages you have to use `Magento_Customer/js/customer-data` and work with it as I've shown in prev lectures